packages/trace-viewer/index.html

```diff
@@ -30,9 +30,14 @@
       <p>The Playwright Trace Viewer must be loaded over the <code>http://</code> or <code>https://</code> protocols.</p>
       <p>For more information, please see the <a href="https://aka.ms/playwright/trace-viewer-file-protocol">docs</a>.</p>
     </dialog>
+    <dialog id="fallback-error-no-service-worker">
+      <p>The Playwright Trace Viewer requires service workers to be enabled.</p>
```

Comment:
could this also include a check for whether the browser is firefox, which then displays some additional information like

> Private browsing on Firefox does not support service workers, see https://bugzilla.mozilla.org/show_bug.cgi?id=1320796